src/pybind/mgr/orchestrator.py

```diff
@@ -406,6 +406,15 @@ def __init__(self):
         # justify having this field here.
         self.container_id = None
 
+        # Some services can be deployed in clusters. For example, mds's can
+        # have an active and standby daemons, and nfs-ganesha can run daemons
+        # in parallel. This tag refers to the cluster of daemons as a whole.
+        #
+        # For instance, a cluster of mds' all service the same fs, and they
+        # will all have the same cluster_id (which may be the Filesystem name
+        # in the FSMap).
+        self.cluster_id = None
```

Comment:
maybe `service_cluster_id` would be better to avoid any confusion with the ceph cluster fsid?

Reply:
Good point. I'll do that.